converted-ethereum-tests.txt

```diff
@@ -0,0 +1,4 @@
+src/
```

Comment:
this must be a path to `EOFTests/EIP3540/validInvalid.json` to be parsed for the coverage script

Reply:
ok, is the script public? this is not explain anywhere is the file is empty currently.

Reply:
its in https://github.com/ethereum/execution-spec-tests/blob/main/.github/workflows/coverage.yaml
and https://github.com/winsvega/evmtest_coverage/tree/main/dockerhub

the result shows here in github workflow